main/ui/src/main/resources/fxml/preferences_general.fxml

```diff
@@ -30,7 +30,7 @@
 			<RadioButton fx:id="nodeOrientationRtl" text="Right to Left" alignment="CENTER_RIGHT" toggleGroup="${nodeOrientation}"/>
 		</HBox>
 
-		<CheckBox fx:id="startHiddenCheckbox" text="%preferences.general.startHidden"/>
+		<CheckBox fx:id="startHiddenCheckbox" text="%preferences.general.startHidden" visible="${controller.isTrayMenuSupported}" managed="${controller.isTrayMenuSupported}"/>
```

Comment:
```suggestion
		<CheckBox fx:id="startHiddenCheckbox" text="%preferences.general.startHidden" visible="${controller.trayMenuSupported}" managed="${controller.trayMenuSupported}"/>
```

Note that the "is" or "get" prefix will be appended by JavaFX automatically. Therefore it will look for `isIsTrayMenuSupported` or `getIsTrayMenuSupported`, which fails and evaluates to always `false`.

Reply:
Should be fixed in ef88f11